index.d.ts

```diff
@@ -1122,6 +1131,11 @@ declare module powerbi {
         propertyTypes?: {
             [propertyName: string]: ValueTypeDescriptor;
         };
+
+        /** (Optional) Description of the type of instance that the property pane should show. If it doesn't exist, we assume constant only. */
```

Comment:
'should display'

Reply:
Sure, copy-pasted from here: src/Clients/VisualHostCore/services/serializers/visualObjectInstanceSerializer.ts